.github/workflows/test.yml

```
@@ -0,0 +1,223 @@
+            features: "jemalloc"
+          
+          # Windows x86_64
+          - target: x86_64-pc-windows-msvc
+            runner: windows-latest
+            features: ""
+          - target: x86_64-pc-windows-msvc
+            runner: windows-latest
+            features: "jemalloc"
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)